apps/server/src/utils/coerceType.ts

```diff
@@ -1,5 +1,16 @@
 import { isColourHex } from 'ontime-utils';
 
+//TODO: write tests
```

Comment:
:D

I see from the other functions here that this TODOS get forgotten, do you mind adding some simple tests?